pkg/webhook/controlplane/ensurer.go

```diff
@@ -239,6 +239,8 @@ StartLimitIntervalSec=0
 [Service]
 Type=oneshot
 ExecStart=/opt/bin/update-resolv-conf.sh
+RestartForceExitStatus=10
```

Comment:
A better approach would be to pick a standard exit code such as `EX_CONFIG (78)` here instead of `10`.

See [sysexits(3)](https://man7.org/linux/man-pages/man3/sysexits.h.3head.html) for more details.